Package.swift

```diff
@@ -50,6 +50,10 @@ let package = Package(
         "PythonKit",
         "CTensorFlow",
         .product(name: "Numerics", package: "swift-numerics"),
+      ],
+      swiftSettings: [
+        .define("USING_X10_BACKEND"),
+        .define("DEFAULT_BACKEND_EAGER"),
```

Comment:
I wonder if it's possible to eventually enable non-`DEFAULT_BACKEND_EAGER` builds via SwiftPM?

Reply:
It is possible, but requires an API break; I'd want such a switch to be planned (and communicated) prior to executing.